normation/rudder/roles/manage_rudder_repository/molecule/with_custom_repo/sles.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Rpm repo key must be included
+  ansible.builtin.rpm_key:
+    state: present
+    key: "{{ rpm_key_url }}"
+  register: repo_key
+  failed_when: (repo_key is changed) or (repo_key is failed)
+
+- name: Repository must be correct
+  ansible.builtin.zypper_repository:
+    name: Rudder
+    description: Rudder
+    state: present
+    repo: "https://my_private_repo/rpm/6.2/SLES_{{ ansible_distribution_major_version }}/"
```

Comment:
Where points the value my_private_repo ?

Reply:
People can have a private internal repository

Reply:
But when they do, they may want to configure it themselves. So you are right, it is maybe pointless